generators/pkg/generator/generate.go

```diff
@@ -162,14 +166,15 @@ func GeneratePipeline(github *GitHub) (*v1beta1.Pipeline, error) {
 							Name: "REPO_FULL_NAME",
 							Value: v1beta1.ArrayOrString{
 								Type:      v1beta1.ParamTypeString,
-								StringVal: u.Path,
+								StringVal: strings.TrimPrefix(u.Path, "/"),
 							},
 						},
 						{
 							Name: "SHA",
 							Value: v1beta1.ArrayOrString{
-								Type:      v1beta1.ParamTypeString,
-								StringVal: "$(params.gitrevision)",
+								Type: v1beta1.ParamTypeString,
+								// TODO: the change from "$(params.gitrevision)" can be removed when the param substitution works with finally
```

Comment:
> the change from "$(params.gitrevision)"

Someone reading the code for the first time won't know what this change was unless they dig back through the history. We should point them towards what to change it to in the future, either by linking to an issue or just indicating in the comment what the future value should be.

Let's also include a link to the underlying param substitution issue.